controller/test-files/event/list/ok.bool_single.res.payload.golden.json

```diff
@@ -0,0 +1,66 @@
+{
+  "data": [
+    {
+      "attributes": {
+        "name": "bool_single",
+        "newValue": true,
+        "oldValue": null,
```

Comment:
@kwk  Why do we have an oldValue here? My understanding is that we won't be showing the oldValue unless it has an actual value.

Reply:
`null` is an actual value.

Reply:
No. For a boolean field `null` isn't a valid value.